osu.Game.Tournament/Screens/Board/Components/BoardBeatmapPanel.cs

```diff
@@ -277,7 +292,37 @@ private void updateState()
 
                 if (swapChoice != null)
                 {
-                    swapIcon.FadeInFromZero(duration: 800, easing: Easing.InCubic);
+                    swapIcon.RotateTo(0);
+                    swapIcon.FadeInFromZero(duration: 100, easing: Easing.InCubic);
+                }
+
+                if (trapChoice != null)
+                {
+                    BorderThickness = 0;
+                    Alpha = 1f;
+                    icon.Icon = FontAwesome.Solid.ExclamationCircle;
+                    instructText.Text = $"{choiceText} set a trap!";
```

Comment:
```suggestion
                    instructText.Text = $"Trap set!";
```

Minor language change to make it sound more native.